README.md

```diff
@@ -41,6 +41,11 @@ See License.txt in the project root for license information.
 Portions of this project are licensed under the MIT license.
 See MIT-License.txt in the project root for license information.
 
+Download
+========
+
+Downloads are available on [this site](https://www.microsoft.com/en-us/download/details.aspx?id=30425).
```

Comment:
If we are going to point to the download, we should be noting that this is for the 2.5 version. We don't have a download site (separate) for the current github builds.